LICENSE

```diff
@@ -1,201 +1,201 @@
-                                 Apache License
```

Comment:
WTF? Fix your editor to leave normal line endings `\n`. And throw away your windows